services/streammanager/googlepubsub/googlepubsubmanager.go

```diff
@@ -52,6 +57,7 @@ func NewProducer(destinationConfig interface{}) (*PubsubClient, error) {
 	for _, s := range config.EventToTopicMap {
 		topic := client.Topic(s["to"])
 		topic.PublishSettings.DelayThreshold = 0
+		topic.PublishSettings.Timeout = o.Timeout
```

Comment:
What if `Timeout` is not specified? What is the default behavior in this case?

Reply:
`netClientTimeoutKeys := []string{"Router." + rt.destName + "." + "httpTimeout", "Router." + rt.destName + "." + "httpTimeoutInS", "Router." + "httpTimeout", "Router." + "httpTimeoutInS"}
	config.RegisterDurationConfigVariable(10, &rt.netClientTimeout, false, time.Second, netClientTimeoutKeys...)`
using a 10sec default